plugins/modules/ndo_route_map_policy_multicast.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_route_map_policy_multicast
+short_description: Manage Multicast Route Map Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Multicast Route Map Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+author:
+- Akini Ross (@akinross)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be a tenant template.
+    type: str
+    required: true
+  route_map_policy:
+    description:
+    - The name of the Multicast Route Map Policy.
+    type: str
+    aliases: [ name ]
+  route_map_policy_uuid:
+    description:
+    - The uuid of the Multicast Route Map Policy.
+    - This parameter is required when the O(route_map_policy) needs to be updated.
+    type: str
+    aliases: [ uuid ]
+  description:
```

Comment:
Should we add some aliases for description?

Reply:
For multiple other modules ( see template modules where description ) we do not add an alias, so do not see why we should now introduce here. If so perhaps raise an issue and change it consistently everywhere.

Reply:
Its fine to keep it consistent. I was just thinking about what we usually do in `ansible-aci` for description.

Reply:
Yeah I know there we use `descr`, but I think this is also due to the model in apic using `descr`. Feel free to open an new issue for changing everywhere if you feel this is an improvement.